identity/app/actions/AuthenticatedActions.scala

```diff
@@ -67,6 +67,19 @@ class AuthenticatedActions(
       }
   }
 
+  def permissionRefiner: ActionRefiner[Request, AuthRequest] = new ActionRefiner[Request, AuthRequest] {
+    override val executionContext = ec
+
+    def refine[A](request: Request[A]) =
+      authService.authenticateUserForPermissions(request) match {
+        case Some(permUser) => Future.successful(Right( new AuthenticatedRequest( permUser, request)))
```

Comment:
Unusual spacing on this line.  You need to sort out your auto-formatting settings in Intellij 😄